src/utils/print.jl

```diff
@@ -29,6 +29,7 @@ function Base.show(io::IO, ::MIME"text/html", sys::System)
         sys.data,
         backend = :html,
         tf = PrettyTables.tf_html_simple,
+        standalone = false,
```

Comment:
So we want the same thing on line 53 too?

Reply:
Those are not html-specific. There are two other places, however.

Reply:
That should resolve it! I was commenting from my phone and wasn’t able to read the code properly I think.